.github/workflows/metallb_e2e.yml

```diff
@@ -75,6 +75,9 @@ jobs:
       - name: Deploy Metal LB Operator
         run: |
           IMG=${built_image} KUSTOMIZE_DEPLOY_DIR="config/frr-on-ci/" make deploy
+      - name: Sleep for 30 seconds
```

Comment:
Can you add a comment on why this is required (or even better, add a kubectl wait to wait for the operator's pod to be running)?

Reply:
I think this is not only about pod coming up, webhook has to be started properly as its configured asynchronously in the operator. hence not replacing sleep with kubectl wait.